src/Report/ReportInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MtsBenchmarks\Report;
+
+/**
+ * All reports should contain ways to build title, headers, and results.
+ */
+interface ReportInterface
```

Comment:
This isn't really used right now. YAGNI suggests removing it and then adding it later (if there is a later).